### DISCUSSION TITLE
Watch mode not detecting changes

### DISCUSSION THREAD

**Discussion by briangwaltney** [vitest-dev/vitest#2590] [Q&A]
Sorry for the noob question. I'm just getting started with Vitest and I'm sure I'm making a mistake somewhere. 

I'm using Vitest in a NextJS app and tried to configure as shown in the example.

The tests start and run fine on the command. It will usually detect the first change I make an rerun the tests, but then it stops "watching" and doesn't rerun when I make updates to the code or test.

Is there something about NextJS causing this or my config perhaps?

Config file: 

```
/// <reference types="vitest" />

import { defineConfig } from 'vitest/config'
import react from '@vitejs/plugin-react'

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [react()],
  test: {
    environment: 'jsdom',
    alias: {
      '@/': `${__dirname}/src/`,
    }
  },
  
})
```

**Comment by ilKhr** [comment]
I found it, It`s my problem
https://vitejs.dev/config/server-options.html#server-watch:~:text=Using%20Vite%20on%20Windows%20Subsystem%20for%20Linux%20(WSL)%202
